### ISSUE TITLE
No pedestrian and vehicle segmentation in CityEnviron?

### ISSUE THREAD

**Issue by Kapil-23** [microsoft/AirSim#2461]
Hi all,

I have downloaded CityEnviron from AirSim releases v1.2.2 - Windows.
On running script **capture_ir_segmentation.py** from AirSim >> PythonClient >> computer_vision.
**Getting below result (No Segmentation for Vehicle and Pedestrians)**

![ir_00009](https://user-images.githubusercontent.com/54632955/76828555-3fc40880-6847-11ea-9417-9ce0aa18861c.jpg)
![img_00000](https://user-images.githubusercontent.com/54632955/76828565-45215300-6847-11ea-991d-57fc70bb785f.jpg)
![img_00009](https://user-images.githubusercontent.com/54632955/76828570-46eb1680-6847-11ea-9ead-3fda8fd2acb6.jpg)
![ir_00000](https://user-images.githubusercontent.com/54632955/76828572-4783ad00-6847-11ea-92f9-289a80da39db.jpg)

Kindly respond if any one facing same issue.

**Comment by rajat2004** [comment]
Yes, I've noticed the same issue. This will probably be fixed only with a new AirSim release of which there's no timeline as of now.

**Comment by jonyMarino** [comment]
Hi @rajat2004! do you know if it fixed this?

**Comment by rajat2004** [comment]
Not sure, I haven't tested City since then, Will try to test the latest binaries and post, however I think it might be a problem with the environment itself.
